src/python-fastui/tests/test_forms.py

```diff
@@ -469,3 +469,37 @@ def test_form_textarea_form_fields():
             }
         ],
     }
+
+
+class FormNumbersDefaultStep(BaseModel):
+    size: int
+    cost: float
+
+
+def test_form_numbers_default_step():
```

Comment:
Could we also add a test for the case when we don't add a default step? Thanks!

Reply:
added, now we have an `int` without step, a `float` with a default step, and a `float` with a specified step